### PR TITLE
add justification based on metadata if possible

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

#584 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add justification based on metadata returned from adviser

## Description

I added bodies for PRs and issues to give users more info about the reason that Thoth created them.

### CODE REVIEW

File: kebechet/managers/thoth_advise/messages.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+This has caused your application stack to become unsolvable. Please loosen version constraints or change your runtime \
+environment.
+"""
+
+NEW_RELEASE_PR_BODY = """
+Version {version} of {package} was released. {package} is used in your project so Thoth automatically ran advise on \
+your application stack.
+
+This PR contains an automatic update of Pipfile.lock by Kebechet thoth-advise manager.
```
Comment (pacospace) [inline kebechet/managers/thoth_advise/messages.py:85]: Should we divide this based on the recommendation used?

if user requested `latest` -> make sense to update 
if user requested `performance` -> make sense if the new stack gives more performance
if user requested `security` -> ??
if user requested `stable` -> ??

wdyt?

Comment (KPostOffice) [reply kebechet/managers/thoth_advise/messages.py:85]: We can't guarantee that the new package is included in the update

Comment (pacospace) [reply kebechet/managers/thoth_advise/messages.py:85]: > > We can't guarantee that the new package is included in the update
> 
> I think we can provide pipeline units in adviser that read metadata supplied (e.g. which package was newly released) and will make sure these packages are part of the lock file (if found any). Then, the overall score will decide if the lock file is better than the one already used by the user.

I think make sense, and if the score is lower than the previous one, we should not open PR right?

Comment (fridex) [reply kebechet/managers/thoth_advise/messages.py:85]: +1, yes.

**Review by fridex — APPROVED** [review]
nice :+1:

**Comment by goern** [comment]
this is just blocked by Thoth: Advise still running, feel free to merge it manually...

**Comment by KPostOffice** [comment]
@goern I do not have the power

**Pull request merged**